uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/fragment/CheckoutFragmentContract.kt

```diff
@@ -31,7 +31,8 @@ interface CheckoutFragmentContract {
         fun getBookButtonState(
             isPassengerDetailsVisible: Boolean = false,
             arePassengerDetailsValid: Boolean,
-            isPaymentValid: Boolean
+            isPaymentValid: Boolean,
+            isTermsCheckBoxValid: Boolean = true
```

Comment:
why you provide default value for isTermsCheckBoxValid parameter? You provided default value for useCheckboxOnTermsAndConditions in KarhooUISDKConfiguration and you have function named isTermsCheckBoxValid() that depends on value from Configuration, so why provide default value in 2 places?

Reply:
Here it's a method called from multiple places and didn't want to modify all of them for a special case situation

Reply:
so how do you know that in every unmodified cases value "true" should be passed?

Reply:
btw. now bookButtonState depends on 3 parameters. If in some cases you are passing only isn't it wrong?